kratos/includes/constitutive_law.h

```diff
@@ -900,6 +900,7 @@ class KRATOS_API(KRATOS_CORE) ConstitutiveLaw : public Flags
         const Variable<double>& rDerivativeVariable,
         double& rOutput);
 
+
```

Comment:
Changes in core base class should be reverted. Same in source file.

Reply:
done :)